include/lsst/afw/geom/EdgeRolloffFunctor.h

```diff
@@ -0,0 +1,67 @@
+// -*- lsst-c++ -*-
+
+/* 
+ * LSST Data Management System
+ * Copyright 2015 LSST Corporation.
+ * 
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the LSST License Statement and 
+ * the GNU General Public License along with this program.  If not, 
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+
+/**
+ * \file 
+ * @brief Functor to describe edge rolloff effect in CCDs.
+ */
+
+#ifndef LSST_AFW_GEOM_EDGEROLLOFFFUNCTOR_H
+#define LSST_AFW_GEOM_EDGEROLLOFFFUNCTOR_H
+
+#include "lsst/afw/geom/Functor.h"
+
+namespace lsst {
+namespace afw {
+namespace geom {
+
+/// @brief Stubbs parameterization of edge rolloff effect in LSST CCDs.
```

Comment:
Please expand this doc string, including a description of each parameter and some idea of the functional shape of the result. Also, my understanding is that the function is valid within a specific range of x; please make that range clear somehow, e.g. in the description of xmax or in the general comments. (I'm guessing the range is [0, xmax] but that is just a guess).

Also, what happens if x is outside the range? If this functor raises any exceptions then please list them here using @throw